README.md

```diff
@@ -1,16 +1,18 @@
-> Because I'm no longer using Windows frequently, I'm currently looking for an additional maintainer to assist in the Winfetch development. [#17](https://github.com/lptstr/winfetch/issues/17)
-<br>
+# Posh Winfetch
```

Comment:
I'd prefer if these two lines were removed. The logo alone looks much neater, IMO.

Reply:
OK